README.md

```diff
@@ -1,2 +1,52 @@
-# styelint-config-cssrecipes
-cssrecipes shareable config for stylelint 
+# stylelint-config-cssrecipes
+
+> cssrecipes shareable config for stylelint
+
+## Install
+
+```console
+$ npm install stylelint-config-cssrecipes
+```
+
+## Usage
+
+As a JavaScript library
+
+```
+var fs = require("fs")
+var postcss = require("postcss")
+var cssnext = require("cssnext")
+var cssrecipesConfig = require("stylelint-config-cssrecipes")
+var reporter = require("postcss-reporter")
+var stylelint = require("stylelint")
+
+var input = fs.readFileSync("index.css", "utf8")
+
+var output = postcss()
+  .use(cssnext())
+  .use(stylelint(cssrecipesConfig))
+  .use(reporter())
+  .process(input)
+
+  fs.writeFileSync("dist/index.css", output)
+
+
+```
+
+
+## Contributing
+
+Work on a branch, install dev-dependencies, respect coding style & run tests before submitting a bug fix or a feature.
+
+```console
+$ git clone https://github.com/stylelint/stylelint-config-cssrecipes.git
+$ git checkout -b patch-1
+$ npm install
+$ npm test
```

Comment:
I think there should be some basic tests as the syntax of the config is quite brittle and can easily break. You could grab the ones [stylelint-config-suitcss](https://github.com/stylelint/stylelint-config-suitcss/tree/master/__tests__) if you like.